ansible/roles/sysprep/tasks/main.yml

```diff
@@ -205,3 +205,11 @@
 - name: Disable Swap
   command: swapoff -a
   when: swapon.stdout
+
+- name: Stop and disable firewalld service
```

Comment:
You may need to first check if the Service exists

https://github.com/mesosphere/konvoy/blob/8d174ec0634d2d21728ea457e511de2adc57fa6c/ansible/roles/firewalld/tasks/main.yaml#L2-L9

Reply:
This is a good idea. I'll copy some of the logic there. Good eye.

Reply:
Check to see if the logic added addresses it adequately please